variables.tf

```diff
@@ -194,3 +199,69 @@ variable "bamboo_install_local_chart" {
   type        = bool
   default     = false
 }
+
+
+################################################################################
+# Jira Settings
+################################################################################
+
+variable "jira_db_major_engine_version" {
+  description = "The Postgres major version to use."
+  type        = string
+}
+
+
+variable "jira_db_allocated_storage" {
+  description = "Allocated storage for database instance in GiB."
+  default     = 1000
+  type        = number
+}
+
+variable "jira_db_instance_class" {
+  description = "Instance class of the RDS instance."
+  default     = "db.t3.micro"
+  type        = string
+}
+
+variable "jira_db_iops" {
+  description = "The requested number of I/O operations per second that the DB instance can support."
+  default     = 1000
+  type        = number
+}
+
+variable "jira_helm_chart_version" {
+  description = "Version of Jira Helm chart"
+  type        = string
+  default     = "1.0.0"
```

Comment:
Default version is "1.1.0" now. If this before marging this PR we have Jira Helm chart "1.2.0" released then we should upgrade this to the latest chart version.